### DISCUSSION TITLE
📌 Project Discussion for Future Ready Talent Internship 💥

### DISCUSSION THREAD

**Discussion by Aditya-Bhate** [github/india#160] [Students]
📍 Group project firstly needs a good idea and active contribution from all members ! so...
1. Communication
2. Time management
3. Problem-solving
4. Listening
5. Critical thinking
6. Collaboration
7. Leadership 

Best luck to all ✌ !

**Comment by Aditya-Bhate** [comment]
So firstly is this a group project or not ?

**Reply by scmishra-cse** [reply]
> So firstly is this a group project or not ?

Hi, @Aditya-Bhate No information is given about it so far hence we should have to wait until 10th of OCTOBER. Most probably all participations are individually So I think projects will also be individually (Correct me If I'm Wrong)

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Cyril-Mathew** [comment]
How to create a website using WordPress on the sandbox, an error occurs when I am trying to review and create. I have pasted a screenshot for reference. Please help me with this
![image](https://user-images.githubusercontent.com/70103733/136823918-d266b022-a234-4c0a-ae87-e515d8ae758c.png)

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

@Cyril-Mathew join the community above, we have helped more than 120+ students to complete this 😊

**Comment by Parnika08** [comment]
My lab access is getting suspended gain and again @eymentor3
